docs/ko/docs/tutorial/static-files.md

```diff
@@ -0,0 +1,54 @@
+# 정적 파일
+
+'StaticFiles'를 사용하여 디렉토리에서 정적 파일을 자동으로 제공할 수 있습니다.
+
+## `aiofiles` 설치
```

Comment:
I think this section, **Install `aiofiles`** is deprecated. Please update (fetch) the files to the latest version.